ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.js

```diff
@@ -35,9 +35,6 @@ describe('Workload CVE overview page tests', () => {
             'aria-pressed',
             'true'
         );
-
-        // Check the no filters exist in the chips
-        cy.get(selectors.filterChipGroup).should('not.exist');
```

Comment:
Did you mean to delete this assertion?

Reply:
Yes, this is causing a failure that was masked by the other failures. With the addition of the default filters this check is no longer valid.